libs/ui/src/lib/steps/steps/steps.component.html

```diff
@@ -4,61 +4,81 @@ <h1 *ngIf="title" class="cvi-steps__title" dataAttribute="steps_title">{{ title
+                    <cvi-ng-circle
+                      size="m"
+                      style="--cvi-circle-border-color: var(--cvi-color-sapphire-blue-10); --cvi-circle-color: var(--cvi-color-sapphire-blue-10);"
+                      [progressPercentage]="anyStepSelected ? currentProgressCSSVar : undefined">
+                      {{ i + 1 }}
+                    </cvi-ng-circle>
+                    <span class="cvi-steps__list-item-title">{{ stepTitle }}</span>
+                  </cvi-ng-track>
+                </div>
+                <cvi-ng-track [gap]="4">
+                  <cvi-ng-icon [name]="'arrow_a_right'" class="cvi-steps__list-item-arrow-icon-wrapper"
+                               [svgClass]="'cvi-steps__list-item-arrow-icon'"></cvi-ng-icon>
+                </cvi-ng-track>
               </cvi-ng-track>
-            </div>
-            <div class="cvi-steps__list-item-circle-wrapper--mobile">
-              <cvi-ng-track [gap]="4" [verticalAlignment]="'center'">
-                <cvi-ng-circle
-                  size="m"
-                  style="--cvi-circle-border-color: var(--cvi-color-sapphire-blue-10); --cvi-circle-color: var(--cvi-color-sapphire-blue-10);"
-                  [progressPercentage]="anyStepSelected ? currentProgressCSSVar : undefined">
-                  {{ i + 1 }}
-                </cvi-ng-circle>
-                <span class="cvi-steps__list-item-title">{{ stepTitle }}</span>
+            </button>
+          </li>
+          <li *ngIf="urlStepTitle"
+              class="cvi-steps__list-item cvi-steps__list-item--is-url-step">
+            <button class="cvi-steps__list-item-button" style="background-color: white;" (click)="urlStepClicked()">
+              <cvi-ng-track [layout]="'flex'" [horizontalAlignment]="'justify'" [verticalAlignment]="'center'">
+                <div class="cvi-steps__list-item-circle-wrapper--desktop">
+                  <cvi-ng-track [gap]="2" [verticalAlignment]="'center'">
+                    <cvi-ng-circle [iconName]="'arrow_a_right'"
+                                   [theme]="'light'"
+                                   [severity]="'info'"
+                                   style="--cvi-circle-border-color: var(--cvi-color-sapphire-blue-13)">
+                    </cvi-ng-circle>
+                    <span class="cvi-steps__list-item-title">{{ urlStepTitle }}</span>
+                  </cvi-ng-track>
+                </div>
               </cvi-ng-track>
-            </div>
-            <cvi-ng-track [gap]="4">
-              <cvi-ng-icon [name]="'arrow_a_right'" class="cvi-steps__list-item-arrow-icon-wrapper"
-                           [svgClass]="'cvi-steps__list-item-arrow-icon'"></cvi-ng-icon>
-            </cvi-ng-track>
-          </cvi-ng-track>
+            </button>
+          </li>
+        </ol>
+        <div #rightEdge style="width: 1px"></div>
+        <button *ngIf="isScrollable && !leftEdgeVisible"
```

Comment:
With accessibility/usability in mind, maybe it might make sense to move the scroll buttons in the DOM before the list of the steps (eg line 7, before the start of the `ol` list)? 

This is because they could be more useful that way to a person who can't operate mouse (non-blind). Instead of tabbing through all the step items first and only then getting to the scroll buttons in the tabbing sequence, users could first tab to the scroll buttons, use these to scroll through the step items ("preview" them), and then choose the right step by tabbing through the step list.

Reply:
Yes, when moved buttons before navigation bar, then it was easier to use. Thanks for pointing out. Fix is added.